### DISCUSSION TITLE
PaymentSheet not closing if using the returnURL

### DISCUSSION THREAD

**Discussion by sunmeetsingh2606** [flutter-stripe/flutter_stripe#1582] [Q&A]
Tried this `returnURL: 'flutterstripe://redirect'` but after successful payment it is not closing the payment web window as it usually close without returnURL.

Do you have any ideas for that ?

_Originally posted by @sunmeetsingh2606 in https://github.com/flutter-stripe/flutter_stripe/issues/1487#issuecomment-1867339766_

**Comment by remonh87** [comment]
You need to make sure that the deeplink is accepted by the app see: https://docs.flutter.dev/ui/navigation/deep-linking

**Reply by sunmeetsingh2606** [reply]
Tried this already but it is now working in flutter-stripe payment sheet
